solutions_template/module_template/{{'terraform' if has_terraform_stage}}/stages/{{terraform_stage if has_terraform_stage}}/providers.tf

```diff
@@ -15,18 +15,18 @@
  *
  */
 
+# TODO: Update Terraform providers below.
+
 # Terraform Block
 terraform {
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = ">= 4.65.2"
+      version = ">= 4.50.0"
```

Comment:
Why downgrading the version?

Reply:
Good catch. I guess it was from another module. I'll fix this.